.env.example

```
@@ -14,8 +12,20 @@ TINYAUTH_DISABLEANALYTICS="false"
 TINYAUTH_DISABLEUIWARNINGS="false"
+
+# Logging Configuration
+
+# Log level: trace, debug, info, warn, error
+TINYAUTH_LOG_LEVEL="info"
 # Enable JSON formatted logs
-TINYAUTH_LOGJSON="false"
+TINYAUTH_LOG_JSON="false"
+# Specific Log stream configurations
+# APP and HTTP log streams are enabled by default, and use the global log level unless overridden
+TINYAUTH_LOG_STREAMS_APP_ENABLED="true"
```

dotenv-linter (4.0.0)

[warning] [QuoteCharacter] The value has quote characters (', ")

(QuoteCharacter)